core/app/assets/javascripts/refinery/core.js.erb

```diff
@@ -1,11 +1,12 @@
 $(document).ready(function(){
   // focus first field in an admin form.
   $('form input[type=text]:first').focus();
-
-  init_flash_messages();
```

Comment:
I've seen that it was call 2 times before i remove this call, the first call was : https://github.com/refinery/refinerycms/blob/c28e7afae816328280fb6f511094d8cbf100ebd0/core/app/assets/javascripts/refinery/admin.js.erb#L6